docs/serving/services/creating-services.md

```diff
@@ -1,76 +1,71 @@
 
-## Before you begin
-
-To create a Knative Service, you will need:
-* A Kubernetes cluster with [Knative installed](https://knative.dev/docs/install/index.html).
-* [Custom domains](https://knative.dev/docs/serving/using-a-custom-domain/) set up for Knative Services.
-
-## Creating a Knative Service
-
-To create an application, you need to create a YAML file that defines a Knative Service.
-This YAML file specifies metadata about the application, points to the hosted image of the app and allows the Service to be configured.
-
-This guide uses the [Hello World sample app in Go](https://knative.dev/docs/serving/samples/hello-world/helloworld-go) to demonstrate the structure of a Service YAML file and the basic workflow for deploying an app. These steps can be adapted for your own application if you have an image of it available on Docker Hub, Google Container Registry, or another container image registry.
-
-The Hello World sample app does the following:
+The Hello World sample app works as follows:
 1. Reads an environment variable `TARGET`.
 2. Prints `Hello ${TARGET}!`. If `TARGET` is not defined, it will use `World` as the `TARGET`.
 
+## Prerequisites
+
+To create a Knative service, you will need:
+* A Kubernetes cluster with [Knative Serving installed](../../install).
+* [Custom domains](../using-a-custom-domain/) set up for Knative services.
+
 ### Procedure
 
-1. Create a new file named `service.yaml` containing the following information.
-  ```yaml
-  apiVersion: serving.knative.dev/v1
-  kind: Service
-  metadata:
-   name: helloworld-go
-   namespace: default
-  spec:
-   template:
+1. Create a new file named `service.yaml` containing the following information:
+
+    ```yaml
+    apiVersion: serving.knative.dev/v1
+    kind: Service
+    metadata:
+     name: helloworld-go
+     namespace: default
     spec:
-     containers:
-      - image: gcr.io/knative-samples/helloworld-go
-        env:
-          - name: TARGET
-            value: "Go Sample v1"
-  ```
-  * `apiVersion`: The current Knative version.
-  * `name`(metadata): The name of the application.
-  * `namespace`: The namespace that the application will use.
-  * `image`: The URL to the image of the application.
-  * `name`(env): The environment variable printed out by the sample application.
-
-   > Note: If you’re deploying an image of your own app, update the name of the app and the URL of the image accordingly.
+     template:
+      spec:
+       containers:
+        - image: gcr.io/knative-samples/helloworld-go
+          env:
+            - name: TARGET
+              value: "Go Sample v1"
+    ```
+    * `apiVersion`: The current Knative version.
```

Comment:
Not sure why we call out `apiVersion` but not `kind`. Both of these should be stable in all these examples at this point.

Reply:
idk either, this was already there before I did the edits so I'd prefer not to block the PR on it since I think this whole bit could do with rewriting / reformatting to fit properly what we're doing now with "callouts" - can you maybe add a follow up issue for this?

Reply:
If you don't want to rewrite this, I'm happy to have this be the place where we draw the line on updates.